docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests_toolbelt
+importlib_metadata
+jwt
```

Comment:
This is required in order for the sphinx `autodoc` functionality to work (for example, see the following `no module named 'jwt'` error: https://readthedocs.org/projects/python-driver-for-arangodb/builds/21620433/)